docs/deployment/azure/aca-deployment-azd-in-depth.md

```diff
@@ -144,7 +144,7 @@ This file is how `azd` remembers (on a per environment basis) which services sho
 
 ### Initial deployment
 
-1. In order to deploy the .NET Aspire application, authenticate to Azure AD` to call the Azure resource management APIs.
+1. In order to deploy the .NET Aspire application, authenticate to Azure AD to call the Azure resource management APIs.
```

Comment:
Shouldn't Azure AD here be changed to Microsoft Entra ID?

Reply:
Not to my knowledge, no.